### DISCUSSION TITLE
HW on ProxmoxVM with Docker

### DISCUSSION THREAD

**Discussion by zebede22** [immich-app/immich#12113] [Q&A] [answered]
I have a VM on Proxmox where I collect all media files, I'm running Immich on the same server as well due to easy access to the files.
How can I use hw from Nvidia GPU I have on the host machine?
Proxmox -> VM -> Docker

**Comment by sai80082** [comment, marked as answer]
You need to pass though GPU from host to VM.refer to the guides they should help you do that.
[Proxmox wiki](https://forum.proxmox.com/threads/pci-gpu-passthrough-on-proxmox-ve-8-installation-and-configuration.130218/)
[Reddit](https://www.reddit.com/r/homelab/comments/b5xpua/the_ultimate_beginners_guide_to_gpu_passthrough/)
